aws_sagemaker_studio/sagemaker_neo_compilation_jobs/gluoncv_ssd_mobilenet_studio/gluoncv_ssd_mobilenet_neo_studio.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<p>Add comma and 'the' - "When calling <code>compile()</code> , the user is expected to provide"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2893/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>